readthedocs/analytics/tests.py

```diff
@@ -34,13 +34,19 @@ def test_anonymize_ua(self):
         )
 
     def test_get_client_ip_with_x_forwarded_for(self):
-        
         # only client's ip is present
         request = RequestFactory().get('/')
         request.META['HTTP_X_FORWARDED_FOR'] = '203.0.113.195'
         client_ip = get_client_ip(request)
         self.assertEqual(client_ip, '203.0.113.195')
 
+        # only client's ip is present
+        request = RequestFactory().get('/')
+        ip = '2001:abc:def:012:345:6789:abcd:ef12'
```

Comment:
Where did you get the sample ip?

Reply:
I just created it. You can always test if it's a valid IP (it is) with the `ipaddress` module.